src/components/Files/FileUploadDialog.tsx

```diff
@@ -113,8 +127,25 @@ export default function FileUploadDialog({
             </>
           ) : (
             fileUpload.files.map((file, index) => (
-              <div key={index} className="space-y-2">
-                <div className="flex items-center justify-between gap-2 rounded-md bg-secondary-300 px-4 py-2">
+              <div
+                key={index}
+                className="rounded-lg p-4 shadow-md border-2 border-primary-500 relative"
+              >
+                {fileUpload.files.length > 1 && (
+                  <div
+                    className="absolute top-0 left-0 bg-primary-500 text-white px-3 py-1 text-sm font-bold rounded-tl-lg -mt-px -ml-px"
+                    style={{ zIndex: 1 }}
```

Comment:
1. why `zIndex`? 🤔 
2. even if it's needed, why not just use tailwind's `z-*` class? 🤔

Reply:
other than that, LGTM